.jenkins/caffe2/test.sh

```diff
@@ -98,7 +98,7 @@ fi
 ##############################
 # TensorRT integration tests #
 ##############################
-if [[ $BUILD_ENVIRONMENT == *linux-xenial-cuda9-cudnn7-py3* ]]; then
+if [[ $BUILD_ENVIRONMENT == *py3.5-cuda10.1-cudnn7-ubuntu16.04* ]]; then
```

Comment:
I think we need to update the circle ci settings to enable some new instances...

Hi @kostmo , could you please help our Nvidia friend on this CI related task?